test/e2e/scheduling/priorities.go

```diff
@@ -637,7 +637,7 @@ func createRC(ns, rsName string, replicas int32, rcPodLabels map[string]string,
 
 func getRandomTaint() v1.Taint {
 	return v1.Taint{
-		Key:    fmt.Sprintf("kubernetes.io/scheduling-priorities-e2e-taint-key-%s", string(uuid.NewUUID())),
+		Key:    fmt.Sprintf("kubernetes.io/e2e-scheduling-priorities-%s", string(uuid.NewUUID()[:23])),
```

Comment:
IMO we should prefer keys you can do a simple prefix match on, but admittedly moving this to the front was not important. "-taint-key" seems redundant. This should now be 40 + 23 characters.

I don't think having a valid UUID is important, and we should have enough bits, I think.